src/state.py

```diff
@@ -201,7 +201,7 @@ class State(BaseModel):  # pylint: disable=too-few-public-methods
     """
 
     agent_password: typing.Optional[str] = None
-    cluster_key: str = Field(min_length=16, max_length=16)
+    cluster_key: str = Field(min_length=32, max_length=32)
```

Comment:
The key was 8 and now is 16 why changing the validation size to 32 ?

Reply:
Because of this https://docs.python.org/3/library/secrets.html#secrets.token_hex